### PR TITLE
keylight: convert temperatures into Kelvin

### PR DESCRIPTION
**Pull request by terinjokes**

### CODE REVIEW

File: internal/keylight/temperature.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keylight
+
+import "math"
+
+// UnmarshalTemp unmarshals Elgato API temperatures to Kelvin
+func UnmarshalTemp(elgato int) int {
```
Comment (mdlayher) [inline internal/keylight/temperature.go]: Unexport both, and these can live in device.go at the bottom of the file. And let's go with Convert as mentioned in chat.

File: internal/keylight/temperature_test.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+package keylight_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/mdlayher/flightdeck/internal/keylight"
+)
+
+func TestMarshalTemp(t *testing.T) {
```
Comment (mdlayher) [inline internal/keylight/temperature_test.go]: How about combining both tests so we do a round trip conversion for each case. You can add the cases from below to this function's table.

File: internal/keylight/temperature.go (hunk 1 above)
Comment (terinjokes) [reply internal/keylight/temperature.go]: the devices tests is currently blackbox

**Review by mdlayher — APPROVED** [review]
LGTM!

**Comment by mdlayher** [comment]
As discussed I would like to move this code out into its own repository under the MIT License. @terinjokes for posterity could you approve that here? Thank you!

**Comment by terinjokes** [comment]
That's fine by me.

**Comment by mdlayher** [comment]
Thanks!

**Pull request merged**